cpg_pipes/providers/cpg/status.py

```diff
@@ -25,8 +25,23 @@ class CpgStatusReporter(StatusReporter):
     database Analysis entries.
     """
 
-    def __init__(self, smdb: SMDB):
+    def __init__(self, smdb: SMDB, slack_channel: str | None = None):
+        super().__init__()
         self.smdb = smdb
+        self.slack_channel = slack_channel or os.environ.get('CPG_SLACK_CHANNEL')
+        self.slack_token = os.environ.get('CPG_SLACK_TOKEN')
+        if self.slack_channel and not self.slack_token:
```

Comment:
Do we want this logic here or should this instead be set up in the wrapper (i.e. analysis-runner in our case)? In other places we just rely on the environment variables already being set up. That would also make it more clear how this should be configured (right now there's a lot of different options how to set this up, with a mix of flags, environment variables, and secrets).

We could also consider moving the `slack_message_cmd` implementation to `cpg-utils`, which could be more generally useful to send status updates (and reusable by the `StatusReporter`).